integrationtest/pom.xml

```diff
@@ -43,6 +43,7 @@
                 <!-- Spring Boot 3 requires at minimum JDK 17. Hence, we want to ignore these modules on JDK 11: -->
                 <module>mapper/orm-spring</module>
                 <module>showcase/library</module>
+                <module>spring</module>
```

Comment:
Considering that we already have `mapper/orm-spring`, and that this new module uses Hibernate ORM as well... maybe it would be clearer if this was in `mapper/orm-spring-uberjar` or something like this?